templates/lvl-1/setup.sh.tmpl

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+
+PASSWORD=${PASSWORD:-1234567890}
+# we can override the default token tickers as cli args
+STAKE=${1:-ustake}
+FEE=${2:-ucosm}
+
+
+aud init --chain-id=testing testing
+# staking/governance token is hardcoded in config, change this
+sed -i "s/\"stake\"/\"$STAKE\"/" "$HOME"/.aud/config/genesis.json
+(echo $PASSWORD; echo $PASSWORD) | aucli keys add validator
+# hardcode the validator account for this instance
+echo $PASSWORD | aud add-genesis-account validator "1000000000$STAKE,1000000000$FEE"
+# (optionally) add a few more genesis accounts
+for addr in "$@"; do
```

Comment:
might be nicer for users to pass in a number of `name`s that then get `aud keys add {{name}}` and then finally `add-genesis-account`

Reply:
Also adding some error or warning when the user doesn't pass in args.